include/envoy/server/configuration.h

```diff
@@ -132,6 +132,27 @@ class Main {
    *         counters.
    */
   virtual std::chrono::milliseconds statsFlushInterval() PURE;
+
+  /**
+   * @return the time interval after which we count a nonresponsive thread event
+   * as a "miss" statistic.
```

Comment:
Nit: Add the return type at the beginning of the @return doc. This is more for consistency with other Envoy code.
Nit: Prefer space indenting on line wrap to underneath the first word of the previous line excluding @return.